packages/flame_tiled/lib/src/tiled_component.dart

```diff
@@ -191,4 +192,21 @@ class TiledComponent<T extends FlameGame> extends PositionComponent
         );
     }
   }
+
+  /// Returns a list of all the Atlases that were created for this component.
+  ///
+  /// This method is useful for debugging purposes as it allows developers to
+  /// check how the tilesets were packed into the atlas.
+  ///
+  /// It returns a record with the Atlas key and its image.
+  List<(String, Image)> componentAtlases() {
```

Comment:
Since we are within the component already I would just call this `atlases()`, it should be implicit that it is for the component right?
`component.atlases()` vs `component.componentAtlases()`

Reply:
Makes sense!